mne/beamformer/tests/test_lcmv.py

```diff
@@ -228,6 +251,15 @@ def test_lcmv():
     assert_raises(ValueError, apply_lcmv_raw, raw_proj, filters,
                   max_ori_out='signed')
 
+    # Test if setting reduce_rank to True returns an error with not-yet-
+    # implemented orientation selections
+    assert_raises(NotImplementedError, lcmv, evoked, forward_vol, noise_cov,
+                  data_cov, pick_ori=None, weight_norm='nai', reduce_rank=True,
+                  max_ori_out='signed')
+    assert_raises(NotImplementedError, lcmv, evoked, forward_surf_ori,
+                  noise_cov, data_cov, pick_ori='normal', weight_norm='nai',
+                  reduce_rank=True, max_ori_out='signed')
```

Comment:
why is it not implemented for NAI? the eig_inv is a the same code block as with unit_noise_gain.

also please add a test that if reduce_rank=False you do get the ValueError due to complex number.

Reply:
It is, this tests the orientations that are not implemented (no orientation selection and normal). I'll make sure to make that clearer in the comments